lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -181,15 +181,12 @@ class ReportUIFeatures {
   }
 
   /**
-   * @param {{text: string, icon?: string, onClick: () => void}} opts
+   * @param {{container?: Element, text: string, icon?: string, onClick: () => void}} opts
    */
   addButton(opts) {
     const metricsEl = this._document.querySelector('.lh-audit-group--metrics');
-    // Not supported without metrics group.
-    if (!metricsEl) return;
-
-    let buttonsEl = metricsEl.querySelector('.lh-buttons');
-    if (!buttonsEl) buttonsEl = this._dom.createChildOf(metricsEl, 'div', 'lh-buttons');
+    const containerEl = opts.container || metricsEl;
```

Comment:
report-ui-features doesn't have a root report `el`, and psi has 2 reports on the page (and not even attached to DOM when `installFeatures` is called..) so we need a `container` option to specify where the element should go.

Reply:
this would make a great comment on the `container: ` line up above, just sayin' :D

(I stared at the container `querySelector` being identical to the default for a bit until I scrolled down a smidge further to this comment)